checkov/terraform/checks/resource/aws/SQSQueueEncryption.py

```diff
@@ -11,6 +11,11 @@ def __init__(self):
         categories = [CheckCategories.ENCRYPTION]
         super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
 
+    def scan_resource_conf(self, conf):
```

Comment:
@orevron Please adhere to typing (fix imports if needed)
```suggestion
    def scan_resource_conf(self, conf: Dict[str, Any]) -> CheckResult:
```